### PR TITLE
[PUB-2570] Campaign header loading skeleton

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Add a campaign header loading skeleton
- Add text, button and styles skeleton to shared-components.

## Context & Notes
https://buffer.atlassian.net/browse/PUB-2570
I've created the text and button to extend the BDS components, but I'll suggest we move this in the future to the ui repository as a variation of Text and Button. The advantage of us using the same components with just a skeleton state is that we don't need to create the same structure twice, once the data is loaded we already know it's gonna be placed in the right spot.

## Screenshots (if appropriate)
Cenas2

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/campaign/components/ViewCampaign/Header/index.jsx [hunk 1]
```diff
@@ -28,51 +32,72 @@ const Header = ({
...
       <Title>
-        <Color color={campaignDetails.color} />
-        <Name type="h2">{campaignDetails.name}</Name>
+        <Color
+          color={campaignDetails.color}
+          aria-label="Loading"
+          displaySkeleton={isLoading}
+        />
+        <Name type="h2" aria-label="Loading" displaySkeleton={isLoading}>
+          {campaignDetails.name || '________________________________________'}
```
Comment (karelm) [inline packages/campaign/components/ViewCampaign/Header/index.jsx:42]: um... slightly curious on the `'________________________________________'`

Comment (anafilipadealmeida) [reply packages/campaign/components/ViewCampaign/Header/index.jsx:42]: Wanted to give it a certain width without css, so I thought just using no visible content. However even though I have a user-select: none so it's like it doesn't exist, the aria-label= loading is not enough for people with voiceover, if they go one level down the tree they still hear this 🤔 Any suggestions?

Comment (anafilipadealmeida) [reply packages/campaign/components/ViewCampaign/Header/index.jsx:42]: I can use probably the hidden html attribute or something

Comment (karelm) [reply packages/campaign/components/ViewCampaign/Header/index.jsx:42]: okay, thanks for clarifying @anafilipadealmeida 🙌

Comment (anafilipadealmeida) [reply packages/campaign/components/ViewCampaign/Header/index.jsx]: How does it seem to you now @karelm ?😄

Comment (karelm) [reply packages/campaign/components/ViewCampaign/Header/index.jsx]: much better, thanks @anafilipadealmeida 🙌

**Pull request merged**